plugins/discourse-local-dates/assets/javascripts/discourse/components/discourse-local-dates-create-form.js

```diff
@@ -258,10 +258,22 @@ export default Component.extend({
     ];
   },
 
-  _generateDateMarkup(config, options, isRange) {
-    let text = `[date=${config.date}`;
+  _generateDateMarkup(config, options, isRange, toConfig) {
```

Comment:
I undertand why we originally called this `config` but now that you're passing two separate objects it makes less sense. Instead I suggest `fromDateTime` and `toDateTime`.